### DISCUSSION TITLE
Performance Metrics

### DISCUSSION THREAD

**Discussion by Navaneeth272001** [junyanz/pytorch-CycleGAN-and-pix2pix#1562] [Q&A]
How do I calculate the performance metrics of the Cycle GAN model?

**Comment by junyanz** [comment]
It depends on your applications. Most of the works evaluate realism, diversity, and input-output alignment. 
See more discussion [here](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/issues/1486).
